Automattic/jetpack
# Connect in place: Prefix CSS animation names consistently

This PR uses a consistent `jetpack-` prefix for CSS animation names. It's done in order to prevent potential name collisions since all the `@keyframes` live in the global scope.  

This addressed a feedback left on the following PR: https://github.com/Automattic/jetpack/pull/13553#pullrequestreview-294178418 

#### Testing instructions:






* Go to Jetpack dashboard
* Make sure you're disconnected from wordpress.com
* Initiate Jetpack connection and confirm that the 'Loading' text is pulsating and that the loading spinner below is animated

#### Proposed changelog entry for your changes:

* Add a consistent  `jetpack-` prefix to CSS animation names
